src/lib/ext-activitylog.js

```diff
@@ -15,19 +15,8 @@ class Model {
     this.logs.push(...logs);
   }
 
-  addFilter({ logKey, valueEquals }) {
-    this.filter[logKey].push(valueEquals);
-  }
-
-  removeFilter({ logKey, valueEquals }) {
-    const logKeyIndex = this.filter[logKey].indexOf(valueEquals);
-    if (logKeyIndex > -1) {
-      this.filter[logKey].splice(logKeyIndex, 1);
-    }
-  }
-
-  setFilterKeyword(keyword) {
-    this.filter.keyword = keyword;
+  setFilter({ logKey, valueEquals }) {
+    this.filter[logKey] = valueEquals;
```

Comment:
`valueEquals` is longer an accurate description. Even something like this would be better:

```
setFilter(logKey, filterDescriptor) {
```

Maybe with a JSDoc comment before it to explain what the expected argument types are.